client/layout/universal-navbar-header/li-menu-item.jsx

```diff
@@ -21,6 +21,7 @@ const UniversalNavbarLiMenuItem = ( {
 				href={ urlValue }
 				title={ titleValue }
 				tabIndex="-1"
+				target="_self"
```

Comment:
Have we checked that this will work universally for all menu items? Seems like some of them might still want to go through the Calypso router:

https://github.com/Automattic/wp-calypso/blob/7e64d4147af86073bc62b3ce1f5c2cb4c3904193/client/layout/universal-navbar-header/index.jsx#L199

Reply:
That's a fair point.
I pushed un update to specify the `target="_self"` only to the links that need it.